hangman.rb

```diff
@@ -0,0 +1,255 @@
+require 'artii'
+require 'colorize'
+
+class Hangman
+  attr_accessor :guesses, :word, :hidden_word, :wrongs
+
+  def initialize
+    @guesses = []
+    @word = "" #pick word random from dictionary
+    @hidden_word = []
+    @wrongs = 0
+    @fails = 0
+  end
+
+  def make_hidden_word
+    @word.length.times do @hidden_word << "____  "
+    end
+
+  end
+
+  def choose_word(dictionary)
+    @word = dictionary.sample.upcase
+  end
```

Comment:
I like that you've made this its own method. However, the way your code is currently structured it's possible to create an instance of `Hangman` without calling `choose_word`, in which case none of the rest of your code would work.

A cleaner approach might be to call `choose_word` directly from `initialize`, so that it's guaranteed to always happen. This would make your code just a little bit harder to use incorrectly.